components/mpas-albany-landice/bld/namelist_files/albany_input.yaml

```diff
@@ -1,11 +1,29 @@
 %YAML 1.1
 ---
 ANONYMOUS:
+# In order to use ML, change Tpetra to Epetra in the following line, 
+# and "Preconditioner Type: MueLu" to " Preconditioner Type: ML" several lines below 
   Build Type: Tpetra
```

Comment:
Side note: this is going away, and we might as well remove it now.  See https://github.com/MALI-Dev/E3SM/pull/107

Reply:
Ok. Just remove this these lines? The related section? Both?

Reply:
@mperego -- and possibly here as well?

Reply:
Just remove the comments.
We'll take care of the "Build Type: Tpetra" in a subsequent PR.

Reply:
Got it. Thanks.